packages/block-library/src/navigation/view.js

```diff
@@ -177,6 +177,20 @@ const { state, actions } = store( 'core/navigation', {
 					focusableElements[ focusableElements.length - 1 ];
 			}
 		},
+		initNav() {
+			const { ref } = getElement();
+			callbacks.collapseNav( ref );
+			window.addEventListener( 'resize', () =>
+				callbacks.collapseNav( ref )
+			);
+		},
+		collapseNav( ref ) {
+			if ( window.innerWidth < 600 ) {
```

Comment:
This value was in the CSS before. I wonder if there's a way to make this value hookable, without exposing a new API

Reply:
We could start by moving into `constants.js`?

Reply:
Done in 7bce629540